### PR TITLE
feat(journey-maps): display bounding box info

### PR DESCRIPTION
**Pull request by besartmemeti**

### CODE REVIEW

File: src/components-examples/journey-maps/angular/journey-maps-full/journey-maps-full-example.ts [hunk 1]
```diff
@@ -322,6 +327,18 @@ export class JourneyMapsFullExample implements OnInit, OnDestroy {
     return this.mapCenter as { lng: number; lat: number };
   }
 
+  mapBoundingBoxInfo(): { lngSW: number; latSW: number; lngNE: number; latNE: number } | undefined {
+    if (!this.mapBoundingBox) {
+      return;
+    }
+    return {
```
Comment (sebisteiner) [inline src/components-examples/journey-maps/angular/journey-maps-full/journey-maps-full-example.ts:334]: Wouldn't it be easier to directly return a formatted string here?

**Review by sebisteiner — CHANGES_REQUESTED** [review]

**Review by sebisteiner — APPROVED** [review]
LGTM 👍

File: src/showcase/app/journey-maps/journey-maps.module.ts [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import { PortalModule } from '@angular/cdk/portal';
-import { CommonModule } from '@angular/common';
+import { CommonModule, DecimalPipe } from '@angular/common';
```
Comment (kyubisation) [inline src/showcase/app/journey-maps/journey-maps.module.ts:2]: DecimalPipe is already provided by CommonModule: https://github.com/angular/angular/blob/15.1.4/packages/common/src/pipes/index.ts#L55
This change is unnecessary.

**Review by kyubisation — CHANGES_REQUESTED** [review]
See comment above

File: src/showcase/app/journey-maps/journey-maps.module.ts (hunk 2 above)
Comment (besartmemeti) [reply src/showcase/app/journey-maps/journey-maps.module.ts:2]: It still has to be listed as a provider. The following error occurs when it is not given

image

**Comment by sebisteiner** [comment]
> See comment above

I solved it without using the `DecimalPipe`. Can you please re-review?

**Review by kyubisation — APPROVED** [review]
LGTM 👍 
I just realized, I initially misread the code. You were right to register it as a provider to inject it via DI.
I apologize for that misunderstanding.
`toFixed` however seems to be the more efficient solution here.

**Pull request merged**